src/MentionsInputLayout.spec.ts

```
@@ -13,17 +13,48 @@ import {
+import type { PreparedMentionChildConfig } from './types'
 
 describe('MentionsInputLayout', () => {
   const defaultCaretPosition = { left: 0, top: 0 }
+  const defaultQueryInfo = {
+    childIndex: 0,
+    query: 'a',
+    querySequenceStart: 0,
+    querySequenceEnd: 2,
+  }
+  const defaultMentionQuery = {
+    regex: /(@([\w]*))$/,
```

ESLint

[error] /(@([\w]*))$/ can be optimized to /(@(\w*))$/.

(unicorn/better-regex)